packages/@storefront/flux-capacitor/test/unit/core/adapters/search.ts

```diff
@@ -230,35 +231,63 @@ suite('Search Adapter', ({ expect, stub }) => {
       const max = stub(ConfigAdapter, 'extractMaxRefinements').returns(2);
       const config = stub(Selectors, 'config').returns(conf);
       const navigations: any = [
-        { name: 'Adds non-selected refinement', refinements: [4], selected: [], more: false },
-        { name: 'Adds selected refinements', refinements: [6, 7, 4, 5, 6, 8], selected: [1, 5] },
-        { name: 'Adds selected refinement first', refinements: [8, 9], selected: [1], more: true },
-        { name: 'Limits to max', refinements: [0, 1, 2, 3, 4], selected: [1, 2, 3, 4], more: true },
-        { name: 'Limits to max', refinements: [0, 1, 2, 3, 4], selected: [], more: true },
+        { field: 'Adds non-selected refinement', refinements: [4], selected: [], more: false },
+        { field: 'Adds selected refinements', refinements: [6, 7, 4, 5, 6, 8], selected: [1, 5] },
+        { field: 'Adds selected refinement first', refinements: [8, 9], selected: [1], more: true },
+        { field: 'Limits to max', refinements: [0, 1, 2, 3, 4], selected: [1, 2, 3, 4], more: true },
+        { field: 'Limits to max', refinements: [0, 1, 2, 3, 4], selected: [], more: true },
       ];
+      stub(ConfigAdapter, 'extractToggleNavigations').returns([]);
 
+      // tslint:disable
       expect(Adapter.pruneRefinements(navigations, state)).to.eql([
-        { name: 'Adds non-selected refinement', refinements: [4], selected: [], show: [0], more: false },
-        // tslint:disable-next-line max-line-length
-        { name: 'Adds selected refinements', refinements: [6, 7, 4, 5, 6, 8], selected: [1, 5], show: [1, 5], more: true },
-        { name: 'Adds selected refinement first', refinements: [8, 9], selected: [1], show: [1, 0], more: true },
-        { name: 'Limits to max', refinements: [0, 1, 2, 3, 4], selected: [1, 2, 3, 4], show: [1, 2], more: true },
-        { name: 'Limits to max', refinements: [0, 1, 2, 3, 4], selected: [], show: [0, 1], more: true },
+        { field: 'Adds non-selected refinement', refinements: [4], selected: [], show: [0], more: false, boolean: false },
+        { field: 'Adds selected refinements', refinements: [6, 7, 4, 5, 6, 8], selected: [1, 5], show: [1, 5], more: true, boolean: false },
+        { field: 'Adds selected refinement first', refinements: [8, 9], selected: [1], show: [1, 0], more: true, boolean: false },
+        { field: 'Limits to max', refinements: [0, 1, 2, 3, 4], selected: [1, 2, 3, 4], show: [1, 2], more: true, boolean: false },
+        { field: 'Limits to max', refinements: [0, 1, 2, 3, 4], selected: [], show: [0, 1], more: true, boolean: false },
       ]);
+      // tslint:enable
       expect(max).to.be.calledWithExactly(conf);
       expect(config).to.be.calledWithExactly(state);
     });
 
+    it('should mark which navigations are boolean type', () => {
+      const navigations: any = [
+        { field: 'one', refinements: [2, 4], selected: [], more: false },
```

Comment:
Consider pulling these navigation objects into variables which can be spread and enriched in the assertion below. ie:

```
const nav1 = { ... };
const nav2 = { ... };

const navigations = [nav1, nav2, ...]

...

 expect(Adapter.pruneRefinements(navigations, state)).to.eql([
{ ...nav1, boolean: false },
{ ...nav2, boolean: false },
...
]);
```

Reply:
Addressed, resolving.